<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTerminalWindow</class>
 <widget class="QWidget" name="DataTerminalWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FastECU terminal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>K-Line</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Protocol:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="klineProtocol">
                <property name="toolTip">
                 <string>K-Line protocol</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Speed:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="klineBaudRate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>K-Line baudrate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>DB:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="klineDataBits">
                <property name="toolTip">
                 <string>Data bits</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>SB:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="klineStopBits">
                <property name="toolTip">
                 <string>Stop bits</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Parity:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="klineParity">
                <property name="toolTip">
                 <string>Parity</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Tester:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="klineTesterId">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>K-Line tester ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Target:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="klineTargetId">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>K-Line target ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="klineListen">
                <property name="styleSheet">
                 <string notr="true">QPushButton{background-color:rgb(160, 16, 16); color:rgb(255,255,255);}
QPushButton:checked{background-color:rgb(16, 160, 16);}</string>
                </property>
                <property name="text">
                 <string>Listen</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Message:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLineEdit" name="klineMsgToSend"/>
              </item>
              <item>
               <widget class="QPushButton" name="sendKlineMessage">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>CANbus (J2534)</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Protocol:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="canProtocol">
                <property name="toolTip">
                 <string>CANbus protocol</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Speed:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="canBaudRate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>CANbus baudrate</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>ID:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="canIdLength">
                <property name="toolTip">
                 <string>CANbus frame ID length</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Tester:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="canTesterId">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>CANbus tester ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Target:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="canTargetId">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>40</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>CANbus target ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="canListen">
                <property name="styleSheet">
                 <string notr="true">QPushButton{background-color:rgb(160, 16, 16); color:rgb(255,255,255);}
QPushButton:checked{background-color:rgb(16, 160, 16);}</string>
                </property>
                <property name="text">
                 <string>Listen</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Message:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLineEdit" name="canMsgToSend"/>
              </item>
              <item>
               <widget class="QPushButton" name="sendCanMessage">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Message log:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Log to file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="text_edit">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>600</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
